src/dashboard/src/components/accounts/views.py

```diff
@@ -63,9 +62,6 @@ def add(request):
 
 
 def edit(request, id=None):
-    if get_client_config_value('kioskMode') == 'True':
-        return redirect('main.views.forbidden')
-
```

Comment:
Undocumented and unused. I think this was used at some point when we didn't have non-super users.